client/my-sites/checkout/composite-checkout/test/composite-checkout-thank-you.js

```diff
@@ -449,7 +449,7 @@ describe( 'getThankYouPageUrl', () => {
 			siteSlug: 'foo.bar',
 			cart,
 			getUrlFromCookie,
-			isEligibleForSignupDestination: false,
+			isEligibleForSignupDestinationResult: false,
```

Comment:
Good catch! Let's fix this in the test description too?

Reply:
Ah yes, fixed it there too in [c0924c4](https://github.com/Automattic/wp-calypso/pull/49708/commits/c0924c43ea6c00e6d9e407e8eaabb14148352762).